data/articles/201610/1.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+        "name": "Chris Sevilleja",
+        "twitter": "sevilayha"
+    },
+    "website": {
+        "id": 1,
+        "name": "Scotch IO",
+        "site_url": "scotch.io",
+        "feed_url": ""
+    },
+    "category": {
+        "id": 9,
+        "name": "Laravel",
+        "description": "Articles related to Laravel framework"
+    },
+    "url": "https:\/\/scotch.io\/tutorials\/automatically-format-laravel-database-fields-with-accessors-and-mutators",
+    "title": "Automatically Format Laravel Database Fields with Accessors and Mutators",
+    "published_at": "2015-05-10",
```

Comment:
@jottajorge sorry but I've to reject this PR.

Reason - Monthly magazine includes articles added in that month only. Since this article was published in May 2015, it must go to May 2015 magazine.

However we are managing magazines from August 2016 so anything published before August 2016 can not be included.

**Problem:** There are many good articles published before Aug2016 and we should list them.

**Solution:** Issue #98 should solve this issue. However we must wait until that issues is fixed.